src/main/kotlin/com/atlassian/performance/tools/report/api/TableReporter.kt

```diff
@@ -0,0 +1,85 @@
+package com.atlassian.performance.tools.report.api
+
+import com.atlassian.performance.tools.io.api.ensureParentDirectory
+import com.atlassian.performance.tools.report.api.result.InteractionStats
+import com.atlassian.performance.tools.report.result.PrintableInteractionStats
+import com.atlassian.performance.tools.report.table.AuiTabbedTableFactory
+import com.atlassian.performance.tools.report.table.AuiTableFactory
+import org.apache.logging.log4j.LogManager
+import java.io.File
+
+class TableReporter(
```

Comment:
`MultiCohortTable`?
`DataReporter` is one of the most vague class names I ever wrote :) my bad

Reply:
`CohortsSummaryTable`?